src/main/scala/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.scala

```diff
@@ -58,63 +59,64 @@ object CollectDuplexSeqMetrics {
-                              var cs_count: Long = 0,
-                              var cs_fraction: Double = 0,
-                              var cs_fraction_gt_or_eq_size: Double = 0,
-                              var ss_count: Long = 0,
-                              var ss_fraction: Double = 0,
-                              var ss_fraction_gt_or_eq_size: Double = 0,
-                              var ds_count: Long = 0,
-                              var ds_fraction: Double = 0,
-                              var ds_fraction_gt_or_eq_size: Double =0
+                              var cs_count: Count = 0,
+                              var cs_fraction: Proportion = 0,
+                              var cs_fraction_gt_or_eq_size: Proportion = 0,
+                              var ss_count: Count = 0,
+                              var ss_fraction: Proportion = 0,
+                              var ss_fraction_gt_or_eq_size: Proportion = 0,
+                              var ds_count: Count = 0,
+                              var ds_fraction: Proportion = 0,
+                              var ds_fraction_gt_or_eq_size: Proportion =0
                              ) extends Metric
 
   /**
-    * Metrics that capture information specifically about double-stranded tag families. Since double-stranded
-    * families have contributions from two different strands it is useful to see the distribution of families
-    * across both strands.
+    * Metrics produced by `CollectDuplexSeqMetrics` to describe the distribution of double-stranded (duplex)
+    * tag families in terms of the number of reads observed on each strand.
     *
-    * We refer to the two strands as "ab" and "ba" because we identify the two strands by observing the same pair of
-    * UMIs (A and B) in opposite order (A->B vs B->A). Which strand is AB and which is BA is largely arbitrary, so
+    * We refer to the two strands as `ab` and `ba` because we identify the two strands by observing the same pair of
+    * UMIs (A and B) in opposite order (A->B vs B->A). Which strand is `ab` and which is `ba` is largely arbitrary, so
     * to make interpretation of the metrics simpler we use a definition here that for a given tag family
-    * AB is the sub-family with more reads and BA is the tag family with fewer reads.
+    * `ab` is the sub-family with more reads and `ba` is the tag family with fewer reads.
     *
-    * @param ab_size The number of reads in the larger single-strand tag family for this double-strand tag family
-    * @param ba_size The number of reads in the smaller single-strand tag family for this double-strand tag family
-    * @param count The number of families with the A and B SS families of size ab_size and ba_size
-    * @param fraction The fraction of all double-stranded tag families that have ab_size and ba_size
+    * @param ab_size The number of reads in the larger single-strand tag family for this double-strand tag family.
```

Comment:
-> "The number of reads in the `ab` sub-family (the larger single-strand tag family for this double-strand tag family)."